src/clangparser.cpp

```diff
@@ -234,8 +250,8 @@ void ClangTUParser::parse()
   }
 
   // let libclang do the actual parsing
-  //for (i=0;i<argv.size();i++) printf("Argument %d: %s\n",i,argv[i]);
-  p->tu = clang_parseTranslationUnit(p->index, nullptr,
+  for (i=0;i<argv.size();i++) printf("Argument %d: %s\n",i,argv[i]);
```

Comment:
Looks like a debug statement, should not be present anymore / commented..

Reply:
Commented again.